godot-core/src/obj/script.rs

```diff
@@ -84,6 +84,37 @@ use self::bounded_ptr_list::BoundedPtrList;
 ///         // through create_script_instance().
 ///         create_script_instance(script_instance)
 ///     }
+///   # fn can_instantiate(&self) -> bool { unreachable!() }
+///   # fn editor_can_reload_from_file(&mut self) -> bool { unreachable!() }
+///   # fn get_base_script(&self) -> Option<Gd<Script>> { unreachable!() }
+///   # fn get_global_name(&self) -> StringName { unreachable!() }
+///   # fn inherits_script(&self, _script: Gd<Script>) -> bool { unreachable!() }
+///   # fn get_instance_base_type(&self) -> StringName { unreachable!() }
+///   # unsafe fn placeholder_instance_create(&self, _for_object: Gd<Object>) -> *mut c_void { unreachable!() }
+///   # fn instance_has(&self, _object: Gd<Object>) -> bool { unreachable!() }
+///   # fn has_source_code(&self) -> bool { unreachable!() }
```

Comment:
Doing this mega-list once (in itest) is enough, here we can use a mock like elsewhere:

https://github.com/godot-rust/gdext/blob/6101ea00b324962cc8fd9935d076376c7ac29364/godot-core/src/obj/script.rs#L45-L50

Of course there's a small risk that we run out of sync, but I'd rather have that than constantly breaking CI due to this list changing.